integration_tests/requirements.txt

```diff
@@ -1,7 +1,7 @@
-dbt-snowflake>=1.3.0,<2.0.0
-dbt-bigquery>=1.3.0,<2.0.0
-dbt-redshift>=1.3.0,<1.5.0
-dbt-postgres>=1.3.0,<2.0.0
-dbt-spark>=1.3.0,<2.0.0
-dbt-spark[PyHive]>=1.3.0,<2.0.0
-dbt-databricks>=1.3.0,<2.0.0
\ No newline at end of file
+dbt-snowflake>=1.7.0,<2.0.0
+dbt-bigquery>=1.7.0,<2.0.0
+dbt-redshift>=1.7.0,<2.0.0
+dbt-postgres>=1.7.0,<2.0.0
+dbt-spark>=1.7.0,<2.0.0
+dbt-spark[PyHive]>=1.7.0,<2.0.0
+dbt-databricks>=1.7.0,<2.0.0
```

Comment:
Would you be able to elaborate on why this update is needed? If this is needed across the board, then we should probably change the compatible dbt version for the package.